src/openfl/display/BitmapData.hx

```diff
@@ -651,11 +655,13 @@ class BitmapData implements IBitmapDrawable {
 				
 			}
 			
-			var matrixCache = source.__worldTransform;
+			var matrixCache = Matrix.__pool.get ();
+			matrixCache.copyFrom (source.__worldTransform);
 			source.__updateTransforms (matrix);
 			source.__updateChildren (false);
```

Comment:
Actually, by quickly looking into the code (and even if this code will change drastically soon by the new rendering of OpenFL), would it make sense to call __updateChildren(true) and instead of calling __update inside DisplayObjectContainer.__updateChildren, to just call __updateChildren there (like it is done for SimpleButton)?

Overall, the SimpleButton looks quite different from DisplayObjectContainer when it comes to the missing __traverse and __setWorldTransformInvalid functions and the overloads for __updateTransforms and __setTransformDirty which are not existing for DisplayObjectContainer. To me it feels really inconsistent and maybe it makes sense to rework this, even if we don't use SimpleButton? Of course, this has nothing particularly to do with this PR, but is just an idea ;)

Reply:
Can we leave it for another branch? I don't want to have a lot of unrelated fixes in the same branch.

Reply:
Yes, of course, as said, this has nothing particularly to do with this PR

Reply:
right. sorry i should learn not to comment on important stuff while going off to the party 🎉